allauth/account/signals.py

```diff
@@ -1,9 +1,22 @@
-from django.dispatch import Signal
+from django.contrib.auth.signals import user_logged_out as dj_user_logged_out
+from django.dispatch import Signal, receiver
 
 
 user_logged_in = Signal(providing_args=["request", "user"])
 user_logged_out = Signal(providing_args=["request", "user"])
 
+
+@receiver(dj_user_logged_out, dispatch_uid="forwards_user_logged_out")
+def forwards_user_logged_out(sender, **kwargs):
```

Comment:
If you use another application which uses the django logout function, it means receivers of allauth signal will also receive signals initiated from this other application.
I think it's a better behavior that the allauth' signal is only fired from something which happened in allauth.